manim/__main__.py

```diff
@@ -8,17 +8,16 @@
 import importlib.util
 import types
 
-from . import constants, logger, console, file_writer_config
-from .config.config import camera_config, args
-from .config import cfg_subcmds
-from .utils.module_ops import (
+from manim import constants, logger, console, config, file_writer_config
+from manim import Scene
+from manim.utils.module_ops import (
```

Comment:
Why switch these to absolute?

Reply:
In general, I like to think of the `__main__.py` module as a script that is external to the library that just uses the library functionality. If this is possible, then everything that `__main__` does can be reproduced _and tested_ from within the library itself without having to call the command line entrypoint.

It was already (almost) possible. I switched these imports to absolute to make totally sure that this is possible and continues to be possible in the future.